xml/depl_conf_admin_crowbar.xml

```diff
@@ -1442,13 +1442,13 @@ lrwxrwxrwx 1 0 Jun 19 08:43 eth3 -&gt; ../../devices/pci0000:00/0000:00:1c.0/000
      Apart from the roles listed under <xref linkend="ex.conduits.role"/>
      each node in &cloud; has a unique role, which lets you create modes
      matching exactly one node. The role is named after the scheme
-     <literal>crowbar-d <replaceable>FULLY QUALIFIED
+     <literal>crowbar-<replaceable>FULLY QUALIFIED
      HOSTNAME</replaceable></literal>. The <replaceable>FULLY QUALIFIED
      HOSTNAME</replaceable> in turn is composed of the following: the MAC address of the
```

Comment:
If I understood the bug report correctly, this should be changed to: The FULLY QUALIFIED HOSTNAME for all nodes booted via PXE is composed of the following: a prefix 'd', the MAC address of the network interface used to boot the node via PXE, and the domain name configured on the &admserv;.